sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 1.0.0-beta.3 (Unreleased)
+- Rename enum type `DimensionUnit` to `LengthUnit` on `FormPage`
```

Comment:
nit: I don't think you need to include "on `FormPage`"

Reply:
that was me asking. From the top of my head I couldn't remember where the property lived.
By having the type, our users can easily know if it affects them or not.